live-examples/html-examples/input/button.html

```diff
@@ -0,0 +1,8 @@
+<fieldset>
+    <legend>Button</legend>
+    <form>
+      <p> These buttons work with access key modifiers too. For Chrome, try `Ctrl + Alt + p`(Mac) / `Alt + p`(Others).</p>
+      <input class="btn" type="button" value="Click Me" accesskey="p" onclick="alert('I work with Javascript attached to me. But `button` is preferred over me these days. :( ')">
+      <input class="btn" type="button" value="I am disabled" onclick="alert('I won\'t work' )" disabled>
```

Comment:
Perhaps add another button showing styling of the different states of a button element? See here for example: https://github.com/mdn/interactive-examples/blob/master/live-examples/html-examples/forms/button.html